.chronus/changes/feat-tsp-openapi3-support-allof-2025-4-3-13-30-47.md

```diff
@@ -0,0 +1,9 @@
+---
+changeKind: feature
+packages:
+  - "@typespec/openapi3"
+---
+
+Add support for OpenAPI's 'allOf' schema composition when converting to TypeSpec.
+
+It enables combining multiple schemas into a single type, supporting inheritance and property aggregation.
```

Comment:
[OpenAPI -> tsp] Add support for converting inline schemas using `allOf`

Reply:
Fiexed in commit [3acc08f](https://github.com/microsoft/typespec/pull/7219/commits/3acc08fc408954cfe81326b45e0f5f9f652d17d9)